cms/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+from django.conf import settings
+from os import listdir
+from os.path import isfile, join
+from ckeditor.fields import RichTextField
+
+
+def load_template_choices():
+    return [
+        (f.name, f.name) for f in settings.CMS_TEMPLATE_DIR.iterdir() if f.is_file()
```

Comment:
`(pathlib.Path(__file__).parent / "templates/cms").iterdir()` would be the same without needing a config key